examples/first/main.go

```diff
@@ -55,18 +55,16 @@ func main() {
 			h.ServeHTTP(w, r)
 		})
 	}
-	sig := patron.SIGHUP(func() {
+	sig := func() {
 		fmt.Println("exit gracefully...")
 		os.Exit(0)
-	})
+	}
 
-	srv, err := patron.New(
-		name,
-		version,
-		patron.Routes(routes),
-		patron.Middlewares(middlewareCors),
-		sig,
-	)
+	srv, err := patron.NewBuilder(name, version).
```

Comment:
No need for a new line on each with as long as you do not exceed 120 chars

Reply:
I'm leaving this as is, because it exceeds the 120 chars limit.